packages/docs/docs/getting-started/latex.mdx

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 10
+slug: /latex
+---
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+import CodeBlock from '@theme/CodeBlock';
+import texSource from '!!raw-loader!@motion-canvas/examples/src/scenes/tex';
+
+# Using LaTeX
+We now have a LaTeX component! You can use it as follows:
```

Comment:
We should make a separate section for documenting nodes, outside of "getting started".
We'll add "We now have a LaTeX component!" to the blog post for the new release, but I think we should leave it out of the docs because at some point the node won't be new anymore.

Reply:
I've added a Nodes page. It just has the Latex example right now, but I'm down to open another issue and show some examples of other nodes once this is merged.